src/main.rs

```diff
@@ -47,7 +50,7 @@ async fn main() {
                 .long_flag("serve")
                 .about("Starts the worker that listens for file changes. If another process is already running, this will do it's best to terminate the other process.")
                 .arg(
-                    arg!(--logfile)
+                    arg!(--logfile <FILE>)
```

Comment:
this is neat syntax

Reply:
thanks hehe, but credit goes to clap, they made use of this or value_parser compulsory(otherwise it just treats it as a flag), this was the recommended way for files